setup.py

```diff
@@ -100,7 +100,7 @@ class install_with_pth(install):
     _pth_contents = textwrap.dedent("""
         import os
         var = 'SETUPTOOLS_USE_DISTUTILS'
-        enabled = os.environ.get(var, 'local') == 'local'
+        enabled = os.environ.get(var, 'local') != 'stdlib'
```

Comment:
This behavior exists in two places, here and in _distutils_hack. Probably both places would need to be addressed.